packages/http-server/src/http-server.ts

```diff
@@ -38,6 +38,16 @@ export interface BaseHttpOptions extends ListenOptions {
    * See {@link https://www.npmjs.com/package/stoppable | stoppable}
    */
   gracePeriodForClose?: number;
+  /**
+   * The following are for configuring properties which are directly set on
+   * https://nodejs.org/api/http.html#http_class_http_server and
+   * https://nodejs.org/api/net.html#net_class_net_server
+   */
```

Comment:
Ideally, we should not describe these options ourselves, but inherit them from Node.js typings instead.

However, looks like Node.js typings are not correctly describing these options. Here is the interface I am seeing:

```ts
    interface ServerOptions {
        IncomingMessage?: typeof IncomingMessage;
        ServerResponse?: typeof ServerResponse;
    }
```

IIRC, we are sticking to a pretty old version of `@types/node` (`10.x.y`) to ensure at compile time that we are not accidentally using recent features that are not available by all Node.js versions we support. 

It makes me wonder if upgrading to the latest version would help us?

I think we have enough test coverage to catch any compatibility problems with older Node.js versions.

Thoughts?

Reply:
Please note, Node does not set those properties via constructor options. They are hard-coded defaults, which can be changed by setting a new value.

Earlier `httpServer` was not accessible, now it is. So users can set these properties like this:

```ts
app.restServer.httpServer!.server.keepAliveTimeout = 9000;
```

Maybe we don't need to provide the ability to configure those properties via RestOptions?

Reply:
> Please note, Node does not set those properties via constructor options.

I was not aware of that. In that case, my comment above does not apply.

> Maybe we don't need to provide the ability to configure those properties via RestOptions?

I don't have a strong opinion on that 🤷🏻